commons/src/main/java/io/hyscale/commons/utils/LogProcessor.java

```diff
@@ -46,8 +55,12 @@ public void writeLogFile(InputStream is, String logFile) throws IOException {
 		Files.copy(is, Paths.get(logFile), StandardCopyOption.REPLACE_EXISTING);
 	}
 
-	public TailLogFile tailLogFile(File logFile, TailHandler handler) {
-		if (!logFile.exists()) {
+	public TailLogFile tailLogFile(File logFile, TailHandler handler) throws HyscaleException {
+		if(handler == null){
```

Comment:
Handler is not a mandatory attribute while tailing...

Reply:
but how will it know end of file?